storyblok/components.json

```diff
@@ -1778,18 +1779,43 @@
           "options": [],
           "pos": 4
         },
+        "heading_tag": {
```

Comment:
Just out of curiosity is this file auto generated?

Reply:
Good question! We have a script to fetch components updates (when you add a field in your own demo space in Storyblok CMS during local development). 
### Fetch space

If you update components in Storyblok, this is your way to get them into the repository. Will overwrite
`storyblok/components.json`. Make sure you do this if you want to get a schema change into production or staging!

```bash
bin/fetch-space --space 1337 (The space ID)
```